lib/upsert/merge_function/postgresql.rb

```diff
@@ -116,14 +116,18 @@ def server_version
       end
 
       def schema_query
+        table_name_arguments = table_name.is_a?(Array) ? table_name : ["public", table_name]
```

Comment:
i don't think we should assume `public` is at the front of pg's `search_path` ... it would be better to leave it out

(i don't quite understand what's going on, so maybe the addition of the schema is necessary somehow?)